src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -69,14 +72,75 @@ def make_artifacts(
     "-b",
     "--mark-best",
     is_flag=True,
+    help="Marks this version of results with a 'best' symlink.",
 )
 @click.option(
     "-t",
     "--test-run",
     is_flag=True,
+    help="Skips updating the 'latest' symlink with this version of results.",
 )
 @click.option(
-    "-a", "--artifact_path", type=click.Path(exists=True), default=paths.DEFAULT_ARTIFACT
+    "-i",
+    "--artifact-path",
+    type=click.Path(exists=True),
+    default=paths.DEFAULT_ARTIFACT,
+    show_default=True,
+    help="Path to artifact used in simulation.",
+)
+@click.option(
+    "-j",
+    "--jobmon",
+    is_flag=True,
+    help="Launches a jobmon workflow to generate results.",
+)
+@click.option(
+    "-q",
+    "--queue",
+    type=click.Choice(["all.q", "long.q"]),
+    default="all.q",
+    show_default=True,
+    help="NOTE: only required if --jobmon. "
+    "The cluster queue to assign jobmon tasks to. long.q allows for much "
+    "longer runtimes although there may be reasons to send jobs to that queue "
+    "even if they don't have runtime constraints (eg node availability). ",
+)
+@click.option(
+    "-m",
+    "--peak-memory",
+    type=int,
+    default=200,
```

Comment:
This is a lot of memory? We should be able to decrease this when we actually get parallel, right?

Reply:
Yes, def. This is based on the current memory req'ts for a 1 million popsize